src/applications/vaos/containers/ConfirmationPage.jsx

```diff
@@ -4,34 +4,49 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import {
   getFormData,
-  getChosenFacilityInfo,
   getFlowType,
   getChosenClinicInfo,
+  getChosenFacilityDetails,
 } from '../utils/selectors';
-import { closeConfirmationPage } from '../actions/newAppointment';
-import { FLOW_TYPES } from '../utils/constants';
+import {
+  closeConfirmationPage,
+  fetchFacilityDetails,
+} from '../actions/newAppointment';
+import { FLOW_TYPES, FACILITY_TYPES } from '../utils/constants';
 import ConfirmationDirectScheduleInfo from '../components/ConfirmationDirectScheduleInfo';
 import ConfirmationRequestInfo from '../components/ConfirmationRequestInfo';
 
 export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    if (
+      !this.props.facilityDetails &&
```

Comment:
We might have the details already (from the clinic choice page) and we don't need them for CC, so don't fetch if either of those things are true.